ncm-fstab/src/main/pan/components/fstab/schema.pan

```diff
@@ -8,15 +8,29 @@ include {'quattor/schema'};
 include {'quattor/blockdevices'};
 include {'quattor/filesystems'};
 
-type structure_component_fstab = {
-	include structure_component
-	"protected_mounts" : string[] = list (
+@documentation{ 
+Protected mountpoints and filesystems. 
+If strict protected is true, it won't change an existing entry. Non-strict only keeps it from removal.
+}
+type fstab_protected_entries = {
+    "mounts" : string[] = list (
 	    "/", "/usr", "/home", "/opt", "/var", "/var/lib", "/var/lib/rpm",
 	    "/usr/bin", "/usr/sbin", "/usr/lib", "/usr/local/bin", "/usr/lib64",
 	    "/bin", "/sbin", "/etc", "swap", "/proc", "/sys", "/dev/pts", "/dev/shm",
 	    "/media/floppy", "/mnt/floppy", "/media/cdrecorder", "/media/cdrom",
 	    "/mnt/cdrom", "/boot"
-	)
+    )
+    "filesystems" ? string[]
```

Comment:
Oh - thanks for clarifying that, I had completely mis-interpreted the name. The docs make sense but would fs_types be a better name for this field?